app/src/main/res/layout/activity_edit_todo.xml

```diff
@@ -77,10 +79,6 @@
             android:layout_height="wrap_content"
             app:layout_constraintBottom_toBottomOf="parent">
 
-            <LinearLayout
-                android:layout_width="match_parent"
-                android:layout_height="70dp"
-                android:orientation="horizontal"></LinearLayout>
```

Comment:
This piece of code is necessary for making sure there is enough space for the error messages to appear, specifically that 70dp of height. Without this, when an error appears, the buttons move down the screen to accommodate the message. If you are okay with that behaviour, you can leave it out, but you may want to leave it in.

Reply:
Which error message do you mean? The "Please enter a task name" one? I'll probably add the code back in just to be safe

Reply:
When you're editing a task and try to leave the name blank, or if you enter an invalid date, there is a corresponding Snackbar message that appears.

Reply:
Ok, I put that back in. Thx for explaining, the screenshots were helpful